### ISSUE TITLE
Manager has exited loop because there are no more active jobs. 

### ISSUE THREAD

**Issue by JonesiKC** [swar/Swar-Chia-Plot-Manager#821]
This makes absolutely no sense 

`Manager has exited loop because there are no more active jobs. `

**Comment by winkee01** [comment]
Same here

**Comment by JonesiKC** [comment]
Ok, we have to think more like a common user. Some people may want to prevent plotting the whole disk, so you can set a maximum. Set max_plots: to an high number like 9999999 and in global: max_concurrent: this will work. I have to look into the code if I have time, at the moment I don't have a clue really. but I think max_current in global is the most important as it sais: max_concurrent: The maximum number of plots that your system can run. The manager will not kick off more than this
  #                 number of plots total over time.

**Comment by andyquek** [comment]
Hi, my plot manager auto stopped. Is this the same problem?

**Comment by winkee01** [comment]
> Hi, my plot manager auto stopped even though tem disk isn’t full. Is this the same problem?
> 
> if it is, how do you guys check to confirm the error?

No, you just need to set `max_plots` in your job config, if `max_plots = 3`, then your job will only finish 3 plot and done. If you want it continue to work until hit your disk space, you can set it to a higher number. Note, this option is different from `max_concurrent`.

**Comment by andyquek** [comment]
> > Hi, my plot manager auto stopped even though tem disk isn’t full. Is this the same problem?
> > if it is, how do you guys check to confirm the error?
> 
> No, you just need to set `max_plots` in your job config, if `max_plots = 3`, then your job will only finish 3 plot and done. If you want it continue to work until hit your disk space, you can set it to a higher number. Note, this option is different from `max_concurrent`.

Thanks for clarifying this I’ve searching for this answer for weeks haha
